src/pages/role/AssignRole.tsx

```diff
@@ -91,6 +94,7 @@ export default function AssignRole() {
                     status={user.status}
                     id={user.id}
                     username={user.username}
+                    // onDisableAccount={() => handleDisableAccount(user.id)}
```

Comment:
why this line is commented?

Reply:
Oh i commented it out debugging, it's of no use I should have gotten rid of it.

Reply:
Oh i commented it out debugging, it's of no use I should have gotten rid of it.